calico_cni/policy_drivers.py

```diff
@@ -131,6 +131,14 @@ def generate_tags(self):
         """
         return []
 
+class MesosPolicyDriver(DefaultPolicyDriver):
+    def generate_rules(self):
+        allow = Rule(action="allow")
+        rules = Rules(id=self.profile_name,
+                      inbound_rules=[allow],
```

Comment:
I'm hesitant because it goes against the expectation of multiple CNI networks.  The only reason we do this in Kubernetes is because the Kubernetes networking model explicitly states it.  Is this the case in Mesos?

I think users will want to put a container in one network, another container in another network, and expect them not to be able to talk to each other.